tap_stripe/__init__.py

```diff
@@ -785,13 +804,19 @@ def sync_event_updates(stream_name):
         # The events stream returns results in descending order, so we
         # cannot bookmark until the entire page is processed
         date_window_start = date_window_end
-        date_window_end = date_window_end + date_window_size
+        date_window_end = date_window_end + events_date_window_size
         singer.write_bookmark(Context.state,
                               stream_name + '_events',
                               'updates_created',
                               max_created)
         singer.write_state(Context.state)
 
+    max_created = max(max_created, sync_start_time - events_date_window_size)
```

Comment:
This is correct logic though, can we keep it as start_time of the last date window as discussed?
```suggestion
    max_created = max(max_created, date_window_start )
```